### ISSUE TITLE
[RoadChoice] Documentation: Specify the aspect ratio and min height of the assets for each block

### ISSUE THREAD

**Issue by shirin27** [hlxsites/vg-macktrucks-com#419]
**Description**

This documentation ticket refers to documenting the best practices in terms of the aspect ratio and min height of the assets we use on each block.
This will be documented in the Franklin Library, on each block.

**Acceptance criteria**
- Every block that can have an image added has the image requirements documented in the Franklin Library
- Requirements to be documented per brand

**Comment by shirin27** [comment]
@ZoeBeykirch I added this to the macktrucks repo, but this is something raised by @Kannan-CM and needed for all sites.
Could you please check the priority? Should we add this to a future sprint, but maybe a bit later, after we complete roadchoice, wdyt?

**Comment by ZoeBeykirch** [comment]
@shirin27 Sounds good, I just added this to sprint 11 so we can talk about it more in the next planning session. This may be able to be done in parallel to the Road Choice project but can be a level 3 lower priority

**Comment by BeckyMedlin** [comment]
Added to Road Choice ticket #10 in the new Road Choice repo. @ZoeBeykirch to edit the requirements.
